docs/_static/template_overrides.css

```diff
@@ -31,6 +31,7 @@
 /* Navigation */
 .wy-menu-vertical a, .wy-menu-vertical li.current > a {
   padding-left: 1.45em;
+  color: #666;
```

Comment:
This is awesome, thank you! I didn't notice this because my local copy of the RTD theme (which is better, if only marginally so) was out of date.

I totally agree with your change, in fact what I'd like to suggest is to go a bit further, and also fix the color of the subsection headings by adding this rule:

```
.wy-menu-vertical header, .wy-menu-vertical p.caption {
  color: #555;
}
```

Then, I played around a bit with different shades, and thought that the current color of the headings, `#9b9b9b`, worked well (in my view). It's not quite as contrasty as `#666`, but I think that it helps the selected page pop out a bit more, and it might detract slightly less from the page content. That might, however, be entirely subjective -- I'm blessed with a super-high-quality monitor here, which might not reflect what actual readers see.

What do you think?

Reply:
Looks good to me! 
![image](https://user-images.githubusercontent.com/814322/43036789-5095d106-8cbc-11e8-95b0-1161781691fd.png)

Reply:
Awesome! I don't think I can edit your PR, would you like to update it? (I can also try that when I'm merging things, or add another commit)

Reply:
I already did, almost at the same time as the comment (see commit below this text box) :)

Reply:
Ah, perfect! You're just way too fast for me :-)